packages/SqueakMaps-Core.package/SMAWindow.class/class/bingApiKey..st

```diff
@@ -1,3 +1,4 @@
 accessing
 bingApiKey: anObject
+
 	BingApiKey := anObject
```

Comment:
Parameterbezeichnung `anObject` korrekt?

Reply:
Ich hatte nicht die Zeit, mich in dem Refactor auch noch mit der Struktur des ganzen Projekts auseinanderzusetzen und von jedem Accessor die Klassentypen zu bestimmen.